label_studio/tasks/migrations/0048_failedprediction.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.25 on 2024-08-13 18:36
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0026_auto_20231103_0020'),
+        ('ml_models', '0009_alter_thirdpartymodelversion_provider'),
+        ('ml', '0007_auto_20240314_1957'),
+        ('tasks', '0047_merge_20240318_2210'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FailedPrediction',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('message', models.TextField(blank=True, default='', help_text='The message explaining why generating this prediction failed', null=True, verbose_name='message')),
```

Comment:
My understanding is that it's preferred in django to use null=False, any reason we want to use null here (and for error_type)?

Reply:
https://www.notion.so/humansignal/Django-Database-Migrations-Best-Practices-42109ea03c9543dc84b0bf43e5150ce5?pvs=4#ed560b0240b04a44aa12704e7fa30356

Reply:
Everywhere we have a text field we are using null=True